res/values/themes.xml

```diff
@@ -37,6 +37,9 @@
         <item name="calendar_future_bg_color">@color/calendar_future_bg_color_dark</item>
         <item name="month_other_bgcolor">@color/month_other_bgcolor_dark</item>
         <item name="settings_calendar_offline">@drawable/ic_sync_off_dark</item>
+        <item name="iconAboutAuthors">@drawable/ic_create_light</item>
```

Comment:
The naming of drawables is currently the other way round ;) Light drawables are postfixed with "dark" as they are used in the dark themes.